app/templates/public.hbs

```diff
@@ -7,10 +7,10 @@
         <div class="ui info d-flex">
           {{#if this.model.logoUrl}}
             <LinkTo @route="public">
-              <img src="{{this.model.logoUrl}}" class="logo mr-8" />
+              <img src="{{this.model.logoUrl}}" class="logo mr-8"/>
             </LinkTo>
           {{/if}}
-          <div>
+          <div style="margin-left:220px">
```

Comment:
why inline?

Reply:
Because I knew that those classes are of semantic UI and I did not get any file or folder where the styling of these classes was present hence I did not remove the inline CSS from here